pygbm/splitting.py

```diff
@@ -304,6 +312,31 @@ def split_indices(context, split_info, sample_indices):
     return (sample_indices[:right_child_position],
             sample_indices[right_child_position:])
 
+@njit(parallel=False)
```

Comment:
You can just use `@njit`, parallelism is off by default
```suggestion
@njit
```